cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java

```diff
@@ -480,6 +482,7 @@ protected void configure() {
           bind(QueueClientFactory.class).to(HBaseQueueClientFactory.class).in(Singleton.class);
           bind(QueueAdmin.class).to(HBaseQueueAdmin.class).in(Singleton.class);
           bind(HBaseTableUtil.class).toProvider(HBaseTableUtilFactory.class);
+          bind(DatasetProvider.class).to(DefaultDatasetProvider.class);
```

Comment:
I don't see this change in PR https://github.com/caskdata/cdap/pull/4325. Why is that?